detox/e2e/test/account/about.e2e.ts

```diff
@@ -69,7 +69,6 @@ describe('Account - Settings - About', () => {
         await expect(AboutScreen.databaseSchemaVersionTitle).toHaveText('Database Schema Version:');
         await expect(AboutScreen.databaseSchemaVersionValue).toBeVisible();
         await expect(AboutScreen.copyInfoButton).toBeVisible();
-        await expect(AboutScreen.copyInfoButton).toHaveText('Copy info');
```

Comment:
Why we don't check the text any more?

Reply:
this button has icon, the matcher was failing.
I've added other way of asserting text now.